wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -2148,6 +2186,7 @@ class KotlinGenerator private constructor(
   }
 
   private fun Field.getDeclaration(allocatedName: String) = when {
+    isPacked && isScalar -> CodeBlock.of("var $allocatedName: MutableList<%T>? = null", type!!.typeName)
```

Comment:
```suggestion
    isPacked && isScalar -> CodeBlock.of("var %N: MutableList<%T>? = null", allocatedName, type!!.typeName)
```

(avoid two kinds of string interpolation in one line)